src/client/client.h

```diff
@@ -67,6 +67,7 @@ class NetworkPacket;
 namespace con {
 class Connection;
 }
+using sound_handle_t = int;
```

Comment:
why using and not typedef ? doesn't  this typedef should be in sound.h ?

Reply:
using should be preferred over typedef.  typedef is C and using is C++

Reply:
>doesn't this typedef should be in sound.h ?

It's a using *declaration*. Declarations can be done multiple times in one translation unit as long as they are all the same.
See this as a forward declaration. Like this, we don't need to include sound.h.

Reply:
This `using` should be outside the namespace though, as `using` obeys namespacing rules - unlike `typedef`.

Proof: https://ideone.com/BAB4F5

Reply:
ahhhh, the lack of indentation threw me off